bundles/org.openhab.core.model.thing/src/org/openhab/core/model/thing/internal/GenericItemChannelLinkProvider.java

```diff
@@ -88,6 +89,15 @@ private void createItemChannelLink(String context, String itemName, String chann
         } catch (IllegalArgumentException e) {
             throw new BindingConfigParseException(e.getMessage());
         }
+
+        // Fix the configuration in case a profile is defined without any scope
+        if (configuration.containsKey("profile") && configuration.get("profile") instanceof String profile
+                && profile.indexOf(":") == -1) {
+            configuration.put("profile", ProfileTypeUID.SYSTEM_SCOPE + ":" + profile);
+            logger.debug("Profile for channel {} updated from {} to {}", channelUID, profile,
```

Comment:
IMO this should be a warning, ambiguous configuration should be fixed.
```suggestion
            logger.debug("Profile '{}' for channel '{}' is missing the scope prefix, assuming the correct UID is '{}'. Check your configuration.", profile, channelUID,
```

Reply:
Noone using files is probably providing a prefix so these warnings will annoy a lot of users !
This is more an information than a warning I think.

Reply:
openHAB has evolved over the years and we now use the prefix everywhere, so people should update their configurations. Nothing annoying about that, they deliberately decided for manual configuration. It would agree that it is annoying to  stop supporting the "old style" without a warning.

Reply:
Done